README.md

```diff
@@ -5,17 +5,8 @@ See [wav-lpddr-hw](https://github.com/waviousllc/wav-lpddr-hw) for more details
 about the hardware platform.
 
 ## License
-This project contains two licensing models both falling under the
-GNU GENERAL PUBLIC LICENSE as follows:
-
-* The WAV-LPDDR-SW application code (located under `app` directory) and other
-  portions not explicitly licensed otherwise, are licensed under the
-  GNU GENERAL PUBLIC LICENSE -- see `COPYING` file in this directory for
-  details.
-
-* `libwddr` and the other WAV-LPDDR-SW libraries are licensed under the
-  GNU LESSER GENERAL PUBLIC LICENSE -- see `COPYING.LESSER` file in this
-  directory for details.
+This project is licensed as Apache-2.0. Previous releases used GPL-3.0 and
+LGPL-3.0.
```

Comment:
Do we even have to acknowledge what it was previously under?  Or can we just leave it that it is currently apache?

Reply:
It doesn't have to be there. But I wanted something there incase someone ever came across the non-apache version.

Reply:
Ok, works for me, just wanted to make sure it didn't make it more confusing, but should be ok.